Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -71,7 +72,11 @@ class CredentialsManagerSpec: QuickSpec {
             it("should fail to clear credentials") {
                 expect(credentialsManager.clear()).to(beFalse())
             }
-            
+
+            it("should accept custom keychains") {
```

Comment:
Structure wise I think this makes more sense to wrap this up as part of a `context block` related to passing a custom keychain. Also in this particular test the `toNot` is redundant.